internal/app/machined/pkg/runtime/v1alpha1/platform/platform.go

```diff
@@ -67,6 +67,12 @@ const (
 
 // CurrentPlatform is a helper func for discovering the current platform.
 func CurrentPlatform() (p runtime.Platform, err error) {
+	if _, err := os.Stat("/usr/etc/is-container"); err == nil {
+		fmt.Println("Found /usr/etc/is-container: platform is container")
```

Comment:
I don't think we should leave this message here, as it runs super early in the boot, probably before the log is set up.